ethcore/res/ethereum/foundation.json

```
@@ -4888,6 +4888,105 @@
 				}
 			}
 		},
+		"000000000000000000000000000000000000000a": {
+			"builtin": {
+				"name": "bls12_381_g1_add",
+				"activate_at": "0x7fffffffffffff",
+				"pricing": {
+					"0x7fffffffffffff": {
+						"price": { "bls12_const_operations": { "price": 600 }}
+					}
```

```suggestion
					"bls12_const_operations": { "price": 600 }
```
Either you set
- `activate_at`: `block_number`, and `pricing` > `bls12_const_operations` if there's a single pricing, or
- `pricing` > `block_number` > `price` > `bls12_const_operations` if pricing is expected to change

but not both!

---

This part I didn't change, only fixed names there and activation to be in a distant future as was suggested above.

---

I may have missed this part, but still needs to be fixed.

---

Should be fixed now